notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4449,6 +4449,46 @@ high treason [Lieutenant Dill made a failed attempt to place the bug compromised
 artificial body part <arm> [There was an army recruiter with a bionic right arm up to the elbow.]
 
 
+movie: The Day After Tomorrow (2004)
+====================================
+
+:: Title
+The Day After Tomorrow
+
+:: Description
+The Day After Tomorrow is a 2004 American science fiction disaster film
+directed, co-produced, and co-written by Roland Emmerich. It is based on the
+1999 book The Coming Global Superstorm by Art Bell and Whitley Strieber.
+
+Synopsis: A series of extreme weather events usher in global cooling and lead
+to a new ice age.
+
+:: Date
+2004-05-17
+
+:: References
+https://en.wikipedia.org/wiki/The_Day_After_Tomorrow
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+ice age [The film is principally about Earth suddenly entering into a new ice age.]
+
+:: Major Themes
+human vs. nature [The people of the Northern Hemisphere were brought to the brink by a trio of enormous snow hurricanes.]
```

Comment:
to the brink of what? Yeah I know, but imho better to complete such expressions for clarity.

Reply:
Reworked this motivation to describe the protagonists' experience.